lib/promise-data-to.js

```diff
@@ -9,8 +9,8 @@ const { setHeaders } = require('../lib/set-headers')
 const protocols = { 'http:': require('http'), 'https:': require('https') }
 const SIMULATE = +process.env.SIMULATE
 const MAX_RETRY_ATTEMPTS = Math.abs(process.env.MAX_RETRY_ATTEMPTS) || 3
-const RETRY_TIMEOUT_MS = Math.abs(process.env.RETRY_TIMEOUT_MS) || 500
-const REQUEST_TIMEOUT_MS = Math.abs(process.env.RETRY_TIMEOUT_MS) || 10000
+const RETRY_TIMEOUT_MS = Math.abs(process.env.RETRY_TIMEOUT_MS) || null
```

Comment:
set default values as ```undefined``` rather than ```null``
```suggestion
const RETRY_TIMEOUT_MS = process.env.RETRY_TIMEOUT_MS && Math.abs(process.env.RETRY_TIMEOUT_MS)
const REQUEST_TIMEOUT_MS = process.env.REQUEST_TIMEOUT_MS 
 && Math.abs(process.env.REQUEST_TIMEOUT_MS)
```

Reply:
I think we decided to set it as **null**, since it is a human set "empty" value